lib/api-save-data.js

```diff
@@ -36,6 +36,14 @@ class ApiSaveData extends API {
 		this.dataToSave = apiSaveValidator.validateData();
 
 		this.model = apiSaveValidator.validateModel();
+
+		if(this.postStructValidate) {
+			try {
+				await this.postStructValidate();
+			} catch(e) {
+				throw new ApiSaveError(e, ApiSaveError.codes.INVALID_REQUEST_DATA);
```

Comment:
Deberíamos agregar un nuevo `ErrorCode` para estos casos.

Reply:
INVALID_DATA_VALIDATION ???

Reply:
la realidad es que no es valida la data que mandan.. para el cliente, no importa si es por una validación de estructura o de un get a otra entidad.. la posta es que lo que mandaron es basura.. 

si el error que devolvemos es claro, se debería entender.. pero para mi es in valida la data que mandan... 

sino... que tenían en mente?!

Reply:
El cliente no ve ese code.. Es más para debugging, te diría.

Reply:
bien! se me ocurre

- validation_error
- Invalid_data